Rule value-keyword-case should ignore scss variables

<!-- Please answer the following questions: -->

> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

A bug where a scss font definition variable leads to an unexpected error

> Which rule, if any, is this issue related to?

`value-keyword-case`

> What CSS is needed to reproduce this issue?

```scss
$font-stack: 'Helvetica Neue', Helvetica, sans-serif;
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "value-keyword-case": "lower"
  }
}
```

> Which version of stylelint are you using?

`6.8.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

`gulp-stylelint`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

SCSS variables

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

The following warnings were flagged:

```shell
  15:47  error  Expected "Helvetica" to be "helvetica"  value-keyword-case
  15:58  error  Expected "Arial" to be "arial"          value-keyword-case
  16:31  error  Expected "Georgia" to be "georgia"      value-keyword-case
  16:40  error  Expected "Times" to be "times"          value-keyword-case
```

<br>
I've prepared a failing test and would solve the issue myself as soon as you decide how to proceed.

--- a/src/rules/value-keyword-case/README.md
+++ b/src/rules/value-keyword-case/README.md
@@ -8,6 +8,8 @@ Specify lowercase or uppercase for keywords values.
  *    These values */
 ```
 
+This rule ignores [`<custom-idents>`](https://developer.mozilla.org/en/docs/Web/CSS/custom-ident) of known properties. Values which are paired with non-properties (e.g. `$vars` and custom properties), and do not conform to the primary option, can be ignored using the `ignoreValues: []` secondary option.
+
 ## Options
 
 `string`: `"lower"|"upper"`

